src/prefect/tasks/secrets/vault_secret.py

```diff
@@ -27,9 +27,12 @@ class VaultSecret(SecretBase):
             credentials.
                 Defaults to a PrefectSecret named `VAULT_CREDENTIALS`.
                 Supported vault client authentication methods:
-                * token { 'VAULT_TOKEN: '<token>' }
-                * appRole: { 'VAULT_ROLE_ID': '<role-id>',
-                             'VAULT_SECRET_ID': '<secret-id>' }
+                * token           { 'VAULT_TOKEN: '<token>' }
+                * appRole:        { 'VAULT_ROLE_ID': '<role-id>',
+                                    'VAULT_SECRET_ID': '<secret-id>' }
+                * kubernetesRole: { 'VAULT_KUBE_AUTH_ROLE': '<>',
+                                    'VAULT_KUBE_AUTH_PATH': '<>',
+                                    'VAULT_KUBE_TOKEN_FILE': '<>' }
```

Comment:
Can you include a note that `VAULT_KUBE_TOKEN_FILE` defaults to "/var/run/secrets/kubernetes.io/serviceaccount/token"?